modules/gdscript/gdscript_editor.cpp

```diff
@@ -156,10 +156,16 @@ bool GDScriptLanguage::validate(const String &p_script, int &r_line_error, int &
 	}
 #endif
 	if (err) {
-		GDScriptParser::ParserError parse_error = parser.get_errors().front()->get();
-		r_line_error = parse_error.line;
-		r_col_error = parse_error.column;
-		r_test_error = parse_error.message;
+		if (r_errors) {
+			for (const List<GDScriptParser::ParserError>::Element *E = parser.get_errors().front(); E; E = E->next()) {
+				const GDScriptParser::ParserError &err = E->get();
```

Comment:
This is shadowing this other variable right here:
https://github.com/godotengine/godot/blob/8f67fc58478b037fe01839c13c0c860d0155e5a1/modules/gdscript/gdscript_editor.cpp#L138

Reply:
Thanks @YeldhamDev